src/libraries/Microsoft.Extensions.Http/src/HttpMessageHandlerBuilder.cs

```diff
@@ -61,6 +61,17 @@ public abstract class HttpMessageHandlerBuilder
         /// </returns>
         public abstract HttpMessageHandler Build();
 
+        /// <summary>
+        /// Creates an instance of an <see cref="HttpMessageHandler"/> using the <see cref="DelegatingHandler"/> instances
+        /// provided by <paramref name="additionalHandlers"/>. The resulting pipeline can be used to manually create <see cref="HttpClient"/>
+        /// or <see cref="HttpMessageInvoker"/> instances with customized message handlers.
+        /// </summary>
+        /// <param name="primaryHandler">The primary handler represents the destination of the HTTP message channel.</param>
```

Comment:
I haven't heard "HTTP message channel" expression to be honest... [Our docs here](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpmessagehandler?view=net-5.0) go with "HTTP message handler" term so let's stick with it. I ended up removing the term here completely to use the type instead, but suggest to use it later in `<returns>`
```suggestion
        /// <param name="primaryHandler">An instance of <see cref="HttpMessageHandler"/> to operate at the bottom of the handler chain and actually handle the HTTP transport operations.</param>
```